accelerator/BUILD

```diff
@@ -15,12 +25,23 @@ cc_binary(
         "//conditions:default": ["-DNDEBUG"],
     }),
     deps = [
-        ":apis",
-        ":proxy_apis",
         ":ta_config",
         ":ta_errors",
-        "@served",
-    ],
+        ":http",
+        "@entangled//utils/handles:signal",
+    ] + select({
+        "//connectivity/mqtt:mqtt_enable": [
+            ":message",
+            "//connectivity/mqtt:mqtt_utils",
+            "//utils:cache",
+            "//utils:pow",
+            "@entangled//cclient/api",
```

Comment:
```
//:message
//utils:cache
//utils:pow
@entangled//cclient/api"
```
Those deps have been included in `ta_config`

Maybe we could find the minimal difference between default and MQTT.
Only use select() for necessary deps and files